### ISSUE TITLE
Stacktrace on job detail button

### ISSUE THREAD

**Issue by MartijnBraam** [fsiddi/flamenco#115]
I've just installed flamenco on my server and I'm getting this stacktrace when clicking on the detail button of a running job:

```
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1836, in __call__
    return self.wsgi_app(environ, start_response)
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1820, in wsgi_app
    response = self.make_response(self.handle_exception(e))
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1403, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1477, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1381, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/opt/flamenco/brender/dashboard/application/controllers/jobs.py", line 88, in view_json
    job['average_time'] = seconds_to_time(job['average_time'])
  File "/opt/flamenco/brender/dashboard/application/helpers/__init__.py", line 4, in seconds_to_time
    return str(datetime.timedelta(seconds=seconds))
TypeError: unsupported type for timedelta seconds component: NoneType
```

**Comment by Eibriel** [comment]
Thanks for reporting! The dashboard is not checking if there is actually seconds to  convert.

**Comment by Eibriel** [comment]
Fixed on 23fdf6a05b0dfb950ce3955dfbcfb4bfdc525ff1
